Makefile

```diff
@@ -489,10 +493,24 @@ verify-links:
 	fi;
 .PHONY: verify-links
 
-verify: verify-gofmt verify-codegen verify-crds verify-helm-schemas verify-helm-charts verify-deploy verify-examples verify-govet verify-helm-lint verify-links
+update-scylla-manager-client:
+	@rm -Rf $(MANAGER_CLIENT_DIR) && mkdir -p $(MANAGER_CLIENT_DIR)
+	$(SWAGGER) generate client -A scylla-manager -f $(MANAGER_CLIENT_SRC_DIR) -t $(MANAGER_CLIENT_DIR) &> /dev/null
```

Comment:
nit: this use common values in two places, entire command could be extracted to macro accepting input and output dirs.

Reply:
Extracted command to macro PTAL